src/Blake.BuildTools/Generator/SiteGenerator.cs

```diff
@@ -328,7 +328,7 @@ private static async Task BakeContent(BlakeContext context, GenerationOptions op
 
             // create output filename - remove spaces or dashes, and convert to PascalCase instead
             // Razor filenames must be PascalCase and cannot contain spaces or dashes; this avoids enforcing this convention in markdown files
-            var fileNameParts = fileName.Split([' ', '-'], StringSplitOptions.RemoveEmptyEntries);
+            var fileNameParts = fileName.Split(new char[] { ' ', '-' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Change this back - looks like it was changed during downgrade to .NET 8 which has also been rolled back (which is correct)